packages/components/src/components/mobile-full-page-modal/index.js

```diff
@@ -0,0 +1,4 @@
+import MobileFullPageModal from './mobile-full-page-modal.jsx';
+import './mobile-full-page-modal.scss';
+
+export default MobileFullPageModal;
```

Comment:
Do we need a new component? Is it possible to use existing `MobileDialog` component?

Reply:
@balakrishna-binary 
we use FadeWrapper, PageOverlay, Div100vhContainer, MobileWrapper for each full screen responsive design on Dbot in order to compact codes we create a new modal which contain all of them for responsive design